libs/s25main/mapGenerator/HeadQuarters.h

```diff
@@ -103,7 +103,7 @@ namespace rttr { namespace mapGenerator {
     template<class T_Container>
     void PlaceHeadquarter(Map& map, int index, const T_Container& area)
     {
-        const auto& positions = FindHqPositions(map, area);
+        auto positions = FindHqPositions(map, area);
```

Comment:
Just for the future: I started developing the habit of making everything possible `const` and by now I like it as it avoids accidental misuse (e.g writing to the wrong variable)
Not worth changing now, but for future changes I wanted to suggest adopting this too (although we don't have a requirement for this)

Reply:
It does make sense. I wish there was a shorter way to mark sth as const, like `cauto` for `const auto` :D